### ISSUE TITLE
Search is not accessible for screen readers

### ISSUE THREAD

**Issue by rakelbjort** [doczjs/docz#1358]
## Bug Report

**Describe the bug**

The search input is not accessible, there is no label present

**Expected behavior**

Please consider adding a label describing the input.
You can also hide the label by styling it so it's only readable for a screen reader: 

 {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  border: 0;
}
